src/popup.js

```diff
@@ -0,0 +1,60 @@
+import { createComment, getComments } from './commentAPI.js';
+import { commentCounter } from './counter.js';
+
+const popup = (container, id, title, img, summary, rating, genres, premiered, ended) => {
```

Comment:
- [ ] **Required:** Kindly reposition your design to match with the given wireframe.

Your Design | Template
:---: | :---:
![image](https://user-images.githubusercontent.com/27709832/175287390-c66013ee-0022-4f2d-9176-0cff9cf7a772.png) | ![image](https://user-images.githubusercontent.com/27709832/175286107-c4fa1f41-7a20-4a2f-921d-f21b020c8d5a.png)

- Kindly move your items to the centre.